manuscript.Rmd

```diff
@@ -135,7 +153,6 @@ environment (e.g. `R`, though other common options may be `Rscript` or
 
 This example shows the use of the `--rm` flag to indicate that the
 container should be removed when the interactive session is finished.
-
```

Comment:
good catch, just pushed this to master.

not seeing the create -> created edit in the PR though?